package.yaml

```diff
@@ -50,7 +50,7 @@ library:
 
 tests:
   parser:
-    main: Parser.hs
+    main: parser.hs
```

Comment:
Why the change?

Reply:
Because GHC complains that the module name should be `Parser`, while I want this to be `Main`. That's probably because the tests directory now contains more than one file.